packages/flame/lib/src/collisions/sweep.dart

```diff
@@ -1,18 +1,20 @@
-import 'package:flame/src/collisions/broadphase.dart';
-import 'package:flame/src/collisions/collision_callbacks.dart';
-import 'package:flame/src/collisions/hitboxes/hitbox.dart';
+import 'package:flame/collisions.dart';
 
 class Sweep<T extends Hitbox<T>> extends Broadphase<T> {
   Sweep({super.items});
 
-  final List<T> _active = [];
-  final Set<CollisionProspect<T>> _potentials = {};
+  late final List<T> _active = [];
+  late final Set<CollisionProspect<T>> _potentials = {};
+
+  @override
+  void update() {
+    items.sort((a, b) => (a.aabb.min.x - b.aabb.min.x).ceil());
```

Comment:
Not sure why this has `.ceil()`?

Reply:
I don't remember exactly why I used `ceil`, but it needs to be rounded with one of the rounding methods for the sorting to work.

Reply:
Oh, interesting. I checked the docs for the `sort()` function, and seems like the canonical approach here would be
```suggestion
items.sort((a, b) => a.aabb.min.x.compareTo(b.aabb.min.x));
```

Reply:
Ah right, forgot about that, it's the same in many other oop languages.